cmd/loadtest/loadtest.go

```diff
@@ -1233,6 +1247,84 @@ func loadTestRPC(ctx context.Context, c *ethclient.Client, nonce uint64, ia *Ind
 	return
 }
 
+func loadTestContractCall(ctx context.Context, c *ethclient.Client, nonce uint64) (t1 time.Time, t2 time.Time, err error) {
+	ltp := inputLoadTestParams
+
+	to := ltp.ContractETHAddress
+
+	chainID := new(big.Int).SetUint64(*ltp.ChainID)
+	privateKey := ltp.ECDSAPrivateKey
+
+	tops, err := bind.NewKeyedTransactorWithChainID(privateKey, chainID)
+	if err != nil {
+		log.Error().Err(err).Msg("Unable create transaction signer")
+		return
+	}
+
+	amount := big.NewInt(0)
+	if *ltp.ContractCallPayable {
```

Comment:
I was just testing and thought that maybe we could remove this flag? Let's say the user specified `--mode contract-call --contract-address <...> --calldata <...>` and also `--eth-amount`, then you know you're going to send a transaction with some eth so maybe `--contract-call-payable` is a bit redundant here?

Reply:
Okay makes sense :)